source/includes/steps-enable-authentication-in-shardcluster-nodowntime-transition-config.yaml

```diff
@@ -0,0 +1,95 @@
+stepnum: 1
+level: 4
+title: "Add ``transitionToAuth: true`` to the ``mongod`` configuration file."
+ref: modify-config-files
+pre: |
+  
+  Copy the existing {{program}} configuration file, giving it a distinct
+  name such as ``<filename>-secure``.
+  
+  You will use this new configuration file to transition the {{program}} to
+  enforce authentication in the sharded cluster. Retain the original
+  configuration file for backup purposes.
+
+  Add the following settings to the new configuration file.
```

Comment:
Add :setting:`security.transitionToAuth` and `security.keyFile` settings to the new configuration file.

Replace the table with the yaml code block.

Keep the If using a different internal authentication ... blurb.